shap/explainers/tree.py

```diff
@@ -46,12 +46,22 @@ class TreeExplainer(Explainer):
     implementations either inside the externel model package or in the local compiled C extention.
     """
 
-    def __init__(self, model, **kwargs):
+    def __init__(self, model, direct = False, output_type = "margin", 
+                 ref_X = None, **kwargs):
+        """ 
+        Parameters
+        ----------
+        model : Several sklearn, xgboost, lightbgm, and catboost models are supported.
+        direct : Denotes to directly generate Trees from the xgboost booster model.
+        output_type : Can be "margin", "mse", "logistic", or "logloss".
```

Comment:
I think `model_output` would be a clearer name that highlights how this is connected to the model. Also we need to add checks that flag a problem if someone calls something other than "margin" when doing non-independent tree shap (since it's not supported then).